### PR TITLE
correct the molar mass weight for C10

### PR DESCRIPTION
**Pull request by GitPaean**

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this please

**Comment by GitPaean** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
The code change itself is okay from the narrow point of view of just the implementation file.  Is there a reason why a 10x change in molar mass does not affect any regression tests?

**Comment by GitPaean** [comment]
> The code change itself is okay from the narrow point of view of just the implementation file. Is there a reason why a 10x change in molar mass does not affect any regression tests?

There is no regression tests using it.

File: opm/material/components/C10.hpp [hunk 1]
```diff
@@ -61,7 +61,7 @@ class C10 : public Component<Scalar, C10<Scalar> >
      * \brief The molar mass in \f$\mathrm{[kg/mol]}\f$ of molecular n-Decane.
      */
     static Scalar molarMass()
-    { return 0.0142;}
+    { return 0.142;}
```
Comment (atgeirr) [inline opm/material/components/C10.hpp:64]: Checked with this site: https://www.webqc.org/molecular-weight-of-C10H22.html which claims 142.2817 g/mol. Not sure if that is actually a good place to find such numbers, but if it is I think we should add the extra digits. What would be the canonical source a chemistry expert would use?

Comment (GitPaean) [reply opm/material/components/C10.hpp:64]: This should be no problem. But new question is whether we should add the extra digits to C1, CO2, etc.  Unless you insist strongly, I think correcting the order of the quantity should be what the PR is for.

Comment (atgeirr) [reply opm/material/components/C10.hpp:64]: I agree, merging this now. But please keep this in mind and consider adding more digits to all of them.

**Pull request merged**